package.json

```diff
@@ -44,31 +44,29 @@
   "author": "Anton Golub <a.golub@qiwi.com>",
   "license": "MIT",
   "dependencies": {
-    "tslib": "^2.2.0"
+    "tslib": "^2.4.0"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
-    "@semantic-release/github": "^7.2.3",
-    "@semantic-release/npm": "^7.1.3",
-    "@types/jest": "^26.0.23",
-    "@types/jest-json-schema": "^2.1.3",
-    "coveralls": "^3.1.0",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^8.0.6",
+    "@semantic-release/npm": "^9.0.1",
+    "@types/jest": "^29.2.0",
+    "@types/jest-json-schema": "^6.1.1",
+    "coveralls": "^3.1.1",
     "dts-generator": "^3.0.0",
```

Comment:
`dts-gen` & `flowgen` should be replaced with `@qiwi/libdefkit`. Here's an example: https://github.com/qiwi/blank-ts-repo/blob/master/package.json#L38

es6 bundles should be post-processed via `tsc-esm-fix`:  https://github.com/qiwi/blank-ts-repo/blob/master/package.json#L39